src/components/chart/state-history-charts.ts

```diff
@@ -345,6 +442,12 @@ export class StateHistoryCharts extends LitElement {
     state-history-chart-line {
       width: 100%;
     }
+    .reset-button {
+      position: fixed;
+      bottom: 24px;
+      right: 24px;
```

Comment:
This is something we are adding right now so I don't expect you to know about it but may as well add it

```suggestion
      bottom: calc(24px + var(--safe-area-inset-bottom));
      right: calc(24px + var(--safe-area-inset-right));
```